crates/web5/src/apid/dsa/ed25519.rs

```diff
@@ -28,6 +28,45 @@ impl Ed25519Generator {
     }
 }
 
+pub(crate) fn from_public_key(public_key: &[u8]) -> Result<Jwk> {
```

Comment:
I really appreciate this function and the `extract_public_key` function below, and I really appreciate how they're both `pub(crate)`! and I also appreciate it's inside the `ed25519` module! very thoughtful!

but I think we should call them `public_jwk_from_bytes()` and `public_jwk_extract_bytes()`; this way we enable ourselves down the lines to support more key representations and more primitive types (other than byte arrays) for the input/output. so you could imagine `public_multibase_from_string()`